### DISCUSSION TITLE
Change File Structure of FishNet Package to account for FishNet Plugins.

### DISCUSSION THREAD

**Discussion by Katerlad** [FirstGearGames/FishNet#538] [Feature Request]
When upgrading FishNet it is recommended to delete FishNet folder to do a fresh install of FishNet. 

This unfortunately deletes the plugin folder, where transports are stored and other FishNet Addons. 

Requesting that the file Structure be changed to look like the picture below. 

![image](https://github.com/FirstGearGames/FishNet/assets/54480322/01e0472b-37b9-463e-b66e-9c90aeb53f21)

Then in the Install instructions that I have implimented in the Docs, will say when upgrading to delete the "Core" folder instead. 

We can call that folder whatever makes the most sense!

The Resources folder is for a separate Idea of storing the DefaultPrefabsObject inside, so its easily loadable inside the Network Manager. Will create separate Suggestion for that!

**Comment by micah686** [comment]
It would also help to move the Demos as well to a separate folder

**Comment by FirstGearGames** [comment]
Might make more sense to move default plugins to FishNet Runtime, so we can just tell users to delete everything but plugins.

**Comment by FirstGearGames** [comment]
This was resolved recently. FishNet plugins are now under Runtime\Plugins.
Third party ones are now FishNet\Plugins. Now users can delete everything but the FishNet\Plugins folder for a clean wipe.

**Reply by Katerlad** [reply]
Still might make sense to move everything that is deletable under one folder for simplicity. Less prone to error and easier direction, but this works as well.
